src/relay/pass/quantize_rewrite.cc

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
@tqchen Should l call this file qnn_rewrite.cc to have a clear marking between `quantize` (Quantization in Relay) and qnn (creating new quantized ops) work?